chef-config/lib/chef-config/path_helper.rb

```diff
@@ -55,7 +55,7 @@ def self.path_separator
       end
     end
 
-    path_seperator_regex = [Regexp.escape(File::SEPARATOR), Regexp.escape(path_separator)].uniq.join
+    path_separator_regex = [Regexp.escape(File::SEPARATOR), Regexp.escape(path_separator)].uniq.join
```

Comment:
I can never spell this word correctly 😭

Reply:
I had to google it to make sure I corrected it the right way...